src/packaging/set_commit_in_version.py

```diff
@@ -12,25 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Test suite for ROS."""
+"""This script should be called when creating the nightly builds to add the commit id to the Webots version."""
 
-import glob
 import os
 import sys
-import multiprocessing
-
-from command import Command
 
 path = '..'
 if 'WEBOTS_HOME' in os.environ:
     path = os.environ['WEBOTS_HOME']
 
-command = Command('make -C %s distrib -j%d' % (path, multiprocessing.cpu_count()))
-command.run(silent=False)
-if command.returncode != 0:
-    raise RuntimeError('Error when executing the Make command')
-
-if sys.platform.startswith('linux'):
-    assert len(glob.glob(path + os.sep + 'distribution' + os.sep + '*')) > 2
+if len(sys.argv) != 2:  # no commit id passed as an argument
+    sys.exit('Commit id not passed as argument.')
 else:
-    assert len(glob.glob(path + os.sep + 'distribution' + os.sep + '*')) > 1
+    commit = sys.argv[1]
+    content = ''
+    with open(os.path.join(path, 'src', 'webots', 'core', 'WbApplicationInfo.cpp'), 'r') as f:
+        content = f.read().replace('R2019b', 'R2019b' + '-commit-' + commit)
```

Comment:
I would also add the date of the nightly build in addition to the commit id.
Also, I would read the current version from `resources/version.txt` instead of hard-coding `R2019b` here.